.changeset/new-coins-eat.md

```diff
@@ -0,0 +1,5 @@
+---
+@nomicfoundation/edr: patch
```

Comment:
Fix before merging: Are you auto-generating this or handwriting changesets? Last time I cut a release it failed to generate the changelog because your changesets don't confirm to the format. It's missing the `""` around the package name

Reply:
Thanks, I'm using the Github bot, but it doesn't know which package changed so I was adding that manually. Will change to CLI.

Reply:
Fixed in [Update new-coins-eat.md](https://github.com/NomicFoundation/hardhat/pull/5137/commits/e6a00e50ff9d94841e1435a66cfa9cca66e97382)

Reply:
The same thing happened to me. I don't know why changesets fails to generate the right package name sometimes.